frontend/src/components/layout/Breadcrumb.tsx

```diff
@@ -0,0 +1,60 @@
+import { useRouterState, Link } from '@tanstack/react-router';
+import { ChevronRight } from 'lucide-react';
+
+function Breadcrumb() {
+  const pathNameMap: { [key: string]: string } = {
+    '/': 'Dashboard',
+    '/sensors': 'Sensoren',
+    '/settings': 'Einstellungen',
+    '/team': 'Mitarbeitende',
+    '/treecluster': 'Baumgruppen',
+    '/vehicles': 'Fahrzeuge',
+    '/waypoints/': 'Einsatzpläne',
+    '/waypoints/new': 'Neuer Einsatzplan',
+    '/map/': 'Kataster',
+  };
```

Comment:
What if we have the route `/treecluster/:id`. Then we need to update the logic here. 

My suggestion, maybe we should move the hole logic from this component to it's own `useBreadcrumb` hook, witch converses the logic and returns the title based on the route.